modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -1,155 +1,99 @@
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-== Basic Architecture
 
-The Runtime Manager exists as:
+You can configure the Runtime Manager agent to export data to external analytics tools.
 
-* a component of the Anypoint Platform.
-* a component of the Anypoint Platform Private Cloud Edition that you can install on-premises.
+Using either the Runtime Manager cloud console or Anypoint Platform Private Cloud Edition, you can:
 
-Anypoint Platform Private Cloud Edition has a Runtime Manager that handles only applications that are deployed on-premises. The cloud-based version of Anypoint Platform has a Runtime Manager that allows you manage applications deployed to xref:index.adoc[CloudHub] as well as applications that are deployed on-premises and to your own cloud servers. When using whichever version of the Runtime Manager, there's always a way to monitor your applications.
-
-Applications deployed to CloudHub can use xref:insight.adoc[Insight]: MuleSoft’s Analytics & Monitoring feature. Applications that you deploy on-premises or to your own cloud servers can be integrated into third-party analytics applications, which is what this document describes.
-
-////
-Applications deployed on Cloud can either use Insights (MuleSoft’s Analytics & Monitoring feature) or be integrated into third party analytics applications for a unified view of monitoring and analytics. Apps deployed on on-prem must be integrated into third party analytics applications.
-////
-
-If you deploy apps from Runtime Manager to customer-managed Mule servers, you can send your Mule app event and analytics data to third party analytics software such as Elk and Splunk.
+* Send Mule event notifications, including flow executions and exceptions, to Splunk or ELK.
++
+See <<Integrating with Splunk>> and <<Integrating with Your ELK Stack>>.
+* Send API Analytics to Splunk or ELK.
++
+See <<Integrating API Analytics with Splunk or ELK>>.
 
-Sending data to third party software is currently not available (out of the box) for applications on xref:cloudhub.adoc[CloudHub].
-One way to achieve this is by using the https://docs.mulesoft.com/runtime-manager/custom-log-appender[CloudHub custom log appender].
 
-Integration between Runtime Manager and third-party analytics software looks like this:
+If you plan to export your payload's contents to be logged, keep in mind that not all formats can be exported.
+See xref:about-logging-of-payload-formats.adoc[Payload Formats and Logging].
 
-image::amc-onprem-diagram-detail.jpg[]
+Sending data to third-party tools is not supported for applications deployed on xref:cloudhub.adoc[CloudHub].
+You can use the CloudHub custom log appender to integrate with your logging system. 
```

Comment:
I had no idea that "appender" was a word! My, my, my...the things you young people come up with!  lol